backend/src/app/filters/customExceptionFilters.ts

```diff
@@ -0,0 +1,24 @@
+import { Catch, ArgumentsHost, HttpException } from '@nestjs/common';
+import { GqlExceptionFilter } from '@nestjs/graphql';
+import { GqlArgumentsHost } from '@nestjs/graphql';
+import { GenericResponse } from '../dto/response/genericResponse';
+
+@Catch()
+export class CustomExceptionFilter implements GqlExceptionFilter {
+  catch(exception: HttpException, host: ArgumentsHost) {
+    const status =
+      exception instanceof HttpException ? exception.getStatus() : 500;
```

Comment:
Can we have all numeric error codes replaced with Http error codes

Reply:
Done